schema/release-schema.json

```diff
@@ -492,7 +492,7 @@
     },
     "Award": {
       "title": "Award",
-      "description": "An award for the given procurement. There can be more than one award per contracting process e.g. because the contract is split among different providers, or because it is a standing offer.",
+      "description": "Decision by the buyer or procuring entity on the supplier with whom it intends to conclude a contract, including the items the supplier should supply and their price. Typically, this decision leads to a contract, but not always (e.g. the award is appealed at court or the supplier refuses to sign the contract). Depending on the jurisdiction, a single decision may concern a single supplier, item and value; or batches of suppliers and/or items and/or values. Similarly, sometimes the award is published as soon as it is made and sometimes only together with a contract (including only being implicitly covered by the contract). As far as possible, the award should be published at the most granular level (i.e. a given supplier will deliver a given item at a given value) and as soon as it is made.",
```

Comment:
@JachymHercher Sorry, I hadn't caught this earlier. But what does "(including only being implicitly covered by the contract)" mean?

Also, if a lot has many items, does the last sentence suggest that each item should be published separately? Should the "i.e." be re-phrased?

Reply:
> @JachymHercher Sorry, I hadn't caught this earlier. But what does "(including only being implicitly covered by the contract)" mean?

I assumed (mainly on the basis of "Source systems can contain data on awards, on contracts, or on both." in https://standard.open-contracting.org/latest/en/guidance/map/awards_contracts_buyers_suppliers/) that there are jurisdictions that don't recognize  "award" as a separate stage in contracting, but just think about contracts. The phrase "Only being implicitly covered by the contract" was supposed to describe this scenario. 
1. I'm not sure this scenario exists? If not, we can remove the sentence.
2. If yes, I agree I should have worded this more clearly. On second thought though, maybe an extra clarification isn't necessary anyway and "only together with a contract" is clear enough? 

> If a lot has many items, does the last sentence suggest that each item should be published separately? Should the "i.e." be re-phrased?
3. It does imply that. Isn't it correct? "As far as possible", don't we want users to separately publish the value for each item / member of the consortium, which then requires an award for each item / member? (If we removed / weakened this significantly, do we say somewhere that e.g. publishing a single award for all lots is a bad practice (outside a framework agreement, see point 5) ?) 

> We need to avoid the "should" in "the items the supplier should supply" which is descriptive (not normative). Maybe rephrase as "the items to be supplied".

4. Why shouldn't this be normative? (Probably linked to the previous point :). )

5. Linked to this discussion, so I'll add it here: in https://github.com/open-contracting/standard/issues/896#issuecomment-755067515 we agreed "That we should recommend/require a single Award that informs about all suppliers in the FA." I'd propose adding it to the definition of award, possibly phrased as an exception to the "as granular as possible".

Reply:
1. Given that we define an award as "Decision by the buyer or procuring entity on the supplier with whom it intends to conclude a contract", I think it is already clear that such a decision must have been made whenever there exists a signed contract, regardless of whether a formal award step preceded the contract's signature.
2. Yes, I don't think the extra clarification is needed, per (1).
3. A contract is related to a single award via `contracts.awardID`. As such, the award cannot be more granular than the eventual contract. The contract, in OCDS, should match as much as possible the real-world contract, so that we have the same number of contracts in OCDS as in the real world, with the same values, etc. The award, on the other hand, is a more abstract concept, and as such can be made more or less granular than the contract. Ideally, it is as granular (but not more granular) than the contract. For example, a lot might have 10 items, and the intention of the buyer is to sign a single contract with the lot's winner, not 10 contracts with the same winner for each item. In that case, there should be one award for that lot (if the procedure is successful, without failed contract signatures, etc.)
4. "the items the supplier should supply" describes a norm established by the real-world contract. It is not a norm established by OCDS. OCDS has no normative power to oblige suppliers to do anything. As such, we need to use a non-normative word instead.
5. Sounds good. Let's work on the wording in #895, since PR comments are less traceable.

Reply:
Points 1,2,4,5 are clear, thank you. Concerning point 3, I'd like to make sure I understand it correctly: 

a. In reality, a single contract can stem from one or more (in case of multiple lots) decisions on which tenderer should get the contract. 

In OCDS 1.X, a single `Contract` follows a single `Award`. Using the lot extension, a single `Award` can refer to multiple lots, so a single contract can cover multiple lots. That's good. However, since some fields are only included in `Award`, this is not flexible enough to always describe reality precisely.  Our (reasonable) appraoch is that when we have to choose between `Contract` or `Award` being closer to reality, we choose `Contract`. For example, when multiple lots (with different values and dates of award) are covered by one contract, then instead of having multiple `Contract`s (each linked to a single award for each lot; each having a value and a date), we stick to one `Contract` and live with not being able to publish information about values of individual lots nor their individual dates of award. Instead, we publish the total value of the award for all lots and *some* date. Do I understand this correctly? If yes, is it documented somewhere, if not, should we add it, probably including guidance on which date to include (e.g. "use the mode / mean date")? 

For OCDS 2.X, do we already have an issue somewhere suggesting Contract:Award being 1:N? (I'm pretty sure there is and I'm pretty sure we alrady discussed this at some point, sorry.)

b. Shouldn't the description of `Award` work better with plurals? In particular, e.g. since an award can concern multiple lots, shouldn't we replace "Decision" by "Decision(s)"?

Reply:
a) Yes, that is correct. In practice, do buyers award multiple lots to a single supplier on different dates? I had assumed they would decide this at once. Not having anywhere to state the awarded value of the lot is indeed an oversight. I've created an issue: https://github.com/open-contracting/ocds-extensions/issues/162 Alternately, not anticipating that lots can be awarded separately and making awards-contracts a 1-n instead of a m-n relationship was an oversight. I added a comment https://github.com/open-contracting/standard/issues/891#issuecomment-832277089

I don't think this is documented anywhere. In any case, it is not in the schema, where it should be. So, yes, let's add it.

For 2.x, there is this issue, which we can re-open: https://github.com/open-contracting/standard/issues/790

b) I guess it depends on our metaphysical perspective on decisions. Is a ham and cheese baguette three decisions, or one? I think it is okay to use the singular for the first word. I worry about confusion if we start with the plural.

Reply:
a) I've double checked with @jskuhrovec and it seems quite common. For example, an award for one of the lots may be delayed compared to the other ones because the company is answering clarification questions from the buyer about its offer, but they want to move forward with the other lots.

Ah, https://github.com/open-contracting/standard/issues/790, right. Indeed. 

b) Funny example :-). I'm wondering whether it isn't subjective/metaphysical just because of a lack of context, while with context, it would be clear cut. In our cases, I think the decision is "choose X amongst multiple options". For sandwiches, this means that if the menu has "ham and cheese baguette", "philadelphia and salmon baguette", "whatever baguette", you are making one choice. If, by contrast, the menu has 1) "bread", "baguette", "bun"; 2) "protein A/B/C" 3) "salad A/B/C" and you want to chose one of each, then it's three decisions. Doesn't chosing a winning bid from multiple bids for the given lot clearly fall within the latter category?

Reply:
b) It's tricky. While standing at the counter, it seems more correct to describe the situation as three decisions, but when a colleague later asks "what did you decide for lunch?", it seems more correct to describe the answer as one decision. The later number is invariable, whether or not additional decisions were introduced at the counter, e.g. "we're out of baguette, will you take a bun?", or a friend says "I thought you were trying to cut down on meat." If the person shared all their micro-decisions in their answer, it might be more correct to describe the answer as multiple decisions, but this would have exceeded the quantity of information that their colleague wanted or expected. ([Maxim of Quantity](https://www.sas.upenn.edu/~haroldfs/dravling/grice.html))

Now, from the perspective of describing the awards in a contracting process, are we (1) at the counter, (2) summarizing our activity for the reader, or (3) over-sharing our activity for the reader?

I think we are at (2). In the case of automated decision making with real-time OCDS publication, we might move to (1), but I consider this case not worth considering. We might be at (3) if we view the disclosure as over-sharing. 

In any case, I think the choice of singular or plural might be best resolved after we decide how to update the rest of the definition to accommodate the award of lots at different times (unless we don't need to?).

Reply:
I have implemented points 1 - 4, see https://github.com/open-contracting/standard/issues/895#issuecomment-888259348. 

Concerning point 5, see https://github.com/open-contracting/standard/issues/896#issuecomment-888254209. 

The only remaining question in this thread is a detail - decision or decision(s). Philosophically, I'm very much on the  "decision(s)" side. Whether in practice that's the case of "too much precision causes confusion" I leave to you :).